sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -76,50 +75,69 @@ export interface ChatClientOptions extends PipelineOptions {
 }
 
 // @public
-export interface ChatMessage extends Omit<RestChatMessage, "senderId" | "content"> {
+export interface ChatMessage {
     content?: ChatMessageContent;
-    sender?: CommunicationUserIdentifier;
+    createdOn: Date;
+    deletedOn?: Date;
+    editedOn?: Date;
+    id: string;
+    senderCommunicationIdentifier?: CommunicationIdentifierModel;
```

Comment:
```suggestion
    sender?: CommunicationIdentifier;
```
The `CommunicationIdentifierModel` is a purely internal type that should never be exposed. Instead use the deserialize function from common and return the `CommunicationIdentifier` type.

For naming, we don't want to put the type of a property into its name, `sender` is a good name. For naming guidelines, I wrote something up:
https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/16334/Identifiers-and-MRIs?anchor=naming